scripts/pylint_extensions.py

```diff
@@ -722,8 +685,6 @@ def visit_importfrom(self, node):
                     node=node,
                     args=(name, modname),
                 )
-            except astroid.AstroidBuildingException:
```

Comment:
AstroidBuildingException is a subclass of AstroidImportError. Hence all the astroid exceptions would be caught in the previous block